docs/tutorials/modify_loss.md

```diff
@@ -0,0 +1,105 @@
+# tutorial 5: Modify Loss
+
+MMDetection provides users with different loss functions. But the default configuration may be not applicable for different datasets or models, so users may want to modify a specific loss to adapt the new situation.
+
+This tutorial first elaborate the computation pipeline of losses, then give some instructions about how to modify each step, the modification can be categorized as tweaking and weighting.
```

Comment:
elaborate -> elaborates, give -> gives.
`, the modification can be categorized as tweaking and weighting.` -> `. The modification can be categorized as tweaking and weighting.`